plugins/redmine_bugs_ruby_lang/lib/redmine_mailing_list_integration/redmine_ext/mailer.rb

```diff
@@ -1,74 +1,49 @@
-module MailingListIntegrationMailer
-
-  def issue_add(user, issue)
-    mailing_lists = issue.project.mail_routes_for_issue(issue)
-
-    record_message(issue, nil, mailing_lists)
-
-    m = super(user, issue)
-
-    m.header[:to] = mailing_lists.map(&:address)
-    m.header[:subject] = "[#{issue.project.name} #{issue.tracker.name}##{issue.id}] #{issue.subject}"
-    m
+Mailer.after_action do
+  if params && (mailing_lists = params[:mailing_lists])
+    if @issue
+      records = mailing_lists.map do |mailing_list|
+        MailingListMessage.create!(mailing_list: mailing_list, issue: @issue, journal: @journal)
```

Comment:
Note: `@issue` and `@journal` are set by Redmine [here](https://github.com/ruby/b.r-l.o/blob/e07916c9068e2ea8d3d3c719ec8f4ab7588f3068/app/models/mailer.rb#L80) and [here](https://github.com/ruby/b.r-l.o/blob/e07916c9068e2ea8d3d3c719ec8f4ab7588f3068/app/models/mailer.rb#L115-L117).